PatientRegistrationApp/src/components/SinglePatientContactInfo.vue

```diff
@@ -1,19 +1,18 @@
-
 <template>
   <v-container fluid>
     <v-row>
       <v-col cols="12" sm="6" md="3">
         <v-text-field
           :rules="['Required']"
           label="Primary Phone Number"
-          placeholder="###-###-####"
+          v-model="phoneNumber"
```

Comment:
Can we be more explicit and say that this is the `patientPhoneNumber` - this could refer to emergency contact on first glance

Reply:
Updated accordingly

Reply:
updated accordingly